infrastructure/sandbox/JITProvisioner/jitprovisioner.tf

```diff
@@ -131,7 +131,7 @@ resource "aws_lambda_function" "jitprovisioner" {
   role                           = aws_iam_role.jitprovisioner.arn
   reserved_concurrent_executions = -1
   kms_key_arn                    = var.kms_key.arn
-  timeout                        = 5
+  timeout                        = 10
```

Comment:
No we need to increase the timeout here?

Reply:
I made this change to match what was live.  This is just to match what the current sandbox behavior is and get it in code

Reply:
Got it, thanks!